locust/stats.py

```diff
@@ -692,9 +698,7 @@ def to_dict(self, escape_string_values=False):
             "max_response_time": proper_round(self.max_response_time),
             "current_rps": self.current_rps,
             "current_fail_per_sec": self.current_fail_per_sec,
-            "median_response_time": self.median_response_time,
```

Comment:
Maybe we should keep median response time? As the code is right now, users don't have any way of adding this statistic if they want to see it

Reply:
I just meant that displaying it by default might not make sense (because it is not a very useful stat, when you have the average...). But I dont really care either way :)